### PR TITLE
Update readme for Kyma CLI

### PR DESCRIPTION
**Pull request by shorim**
changes:

- Added hyperlinks to the `Kyma CLI` repo and the `Kyma` docs.
- Features list are now grouped and promoting the important features of the `Kyma CLI`. Also, new features are added to reflect the changes in `Kyma CLI 2.0`.
- The screenshot is removed, since displaying just the description of the root commands is not meaningful without their corresponding subcommands. Also, it will need to be constantly updated whenever the description of the commands in the `Kyma CLI` changes.

### CODE REVIEW

File: automatic/kyma-cli/README.md [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
-Kyma CLI requires the following software:
-
-* [kubectl](https://github.com/kubernetes/kubectl)
-* [Minikube](https://www.chocolatey.org/packages/Minikube)
-
-## Notes
-
-* This package is automatically updated using the [Chocolatey Automatic Package Update Model (AU)](https://github.com/majkinetor/au/blob/master/README.md).
-  If you find it is out of date by more than a day or two, please contact the maintainer(s) and let them know the package is no longer updating correctly.
+# [<img src="https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/icons/kyma-cli.png" width="48" height="48"/>Kyma CLI - Install, Manage and Test Kyma](<https://chocolatey.org/packages/kyma-cli>)
```
Comment (a-thaler) [inline automatic/kyma-cli/README.md:1]:
```suggestion
# [<img src="https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/icons/kyma-cli.png" width="48" height="48"/>Kyma CLI - Install and Manage Kyma](<https://chocolatey.org/packages/kyma-cli>)

```

File: automatic/kyma-cli/README.md [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
+
+[![GitHub license](https://img.shields.io/github/license/kyma-project/kyma-cli)](https://github.com/kyma-project/cli/blob/master/LICENCE/)
+[![Maintenance status](https://img.shields.io/badge/maintained%3F-yes-green.svg)](https://github.com/dgalbraith/chocolatey-packages/graphs/commit-activity)
+[![AppVeyor build](https://img.shields.io/appveyor/ci/dgalbraith/chocolatey-packages)](https://ci.appveyor.com/project/dgalbraith/chocolatey-packages)
+[![Software version](https://img.shields.io/badge/version-2.0.0-blue)](https://github.com/kyma-project/cli/releases/tags/2.0.0)
+[![Chocolatey package version](https://img.shields.io/chocolatey/v/kyma-cli?label=Chocolatey)](https://chocolatey.org/packages/kyma-cli)
+
+Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install, manage, and test Kyma.
```
Comment (a-thaler) [inline automatic/kyma-cli/README.md:9]:
```suggestion
Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install and manage Kyma.

```

File: automatic/kyma-cli/README.md [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
+* Generates and displays the bash or zsh completion script
+* Launches the Kyma dashboard in a browser window
+* Creates a system on the Kyma cluster with the specified name
+* Deploys Kyma on a cluster based on the current or specified release
+* Imports certificates to local certificates storage or adds domains to the local host file
+* Provisions a new cluster on a platform of your choice. Currently, this command supports cluster provisioning on GCP, Azure, Gardener, and k3d
+* Shows the cluster version and the Kyma CLI version
+* Creates, applies and synchronizes local resources for you Function in the Kyma cluster
+* Runs a Function in Docker from local sources
+* [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests
```
Comment (a-thaler) [inline automatic/kyma-cli/README.md:22]:
```suggestion
* Cluster Management
  * Installs and updates Kyma on a Kubernetes cluster supporting custom configuration
  * Shows the cluster version and the Kyma CLI version
* Local Environment
  * Provisions a local K3D cluster
  * Launches the Kyma dashboard in a browser window
  * Imports certificates to local certificates storage
  * Generates hosts file entries
* Serverless support
  * Creates, applies and synchronizes local resources to Functions in a Kyma cluster
  * Runs a Function in Docker from local sources
```

**Review by a-thaler — CHANGES_REQUESTED** [review]

File: automatic/kyma-cli/README.md [hunk 4]
```diff
@@ -1,32 +1,36 @@
...
+[![AppVeyor build](https://img.shields.io/appveyor/ci/dgalbraith/chocolatey-packages)](https://ci.appveyor.com/project/dgalbraith/chocolatey-packages)
+[![Software version](https://img.shields.io/badge/version-2.0.0-blue)](https://github.com/kyma-project/cli/releases/tags/2.0.0)
+[![Chocolatey package version](https://img.shields.io/chocolatey/v/kyma-cli?label=Chocolatey)](https://chocolatey.org/packages/kyma-cli)
+
+Kyma CLI is a command line tool which supports Kyma developers. It provides a set of commands you can use to install, manage, and test Kyma.
+
+## Features
+
+* Generates and displays the bash or zsh completion script
+* Launches the Kyma dashboard in a browser window
```
Comment (NHingerl) [inline automatic/kyma-cli/README.md:14]:
```suggestion
* Launches the Kyma Dashboard in a browser window

```

File: automatic/kyma-cli/README.md (hunk 2 above)
Comment (NHingerl) [inline automatic/kyma-cli/README.md:9]:
```suggestion
Kyma CLI is a command line tool that supports Kyma developers. It provides a set of commands you can use to install and manage Kyma.

```

File: automatic/kyma-cli/README.md [hunk 5]
```diff
@@ -1,32 +1,36 @@
...
+* Creates a system on the Kyma cluster with the specified name
+* Deploys Kyma on a cluster based on the current or specified release
+* Imports certificates to local certificates storage or adds domains to the local host file
```
Comment (NHingerl) [inline automatic/kyma-cli/README.md:17]: is that either certificates or domains, or can it do both at the same time? If both, then "and"

```suggestion
* Imports certificates to local certificates storage and adds domains to the local host file

```

File: automatic/kyma-cli/README.md [hunk 6]
```diff
@@ -1,32 +1,36 @@
...
+* Provisions a new cluster on a platform of your choice. Currently, this command supports cluster provisioning on GCP, Azure, Gardener, and k3d
```
Comment (NHingerl) [inline automatic/kyma-cli/README.md:18]:
```suggestion
* Provisions a new cluster on a platform of your choice (GCP, Azure, Gardener, and k3d)

```
If we have two full sentences, separated by a full stop, we need a full stop at the end as well. For consistency with the other lines, I'd rephrase it so we have a shorter phrase that can go without full stop.

File: automatic/kyma-cli/README.md [hunk 7]
```diff
@@ -1,32 +1,36 @@
...
+* Shows the cluster version and the Kyma CLI version
+* Creates, applies and synchronizes local resources for you Function in the Kyma cluster
```
Comment (NHingerl) [inline automatic/kyma-cli/README.md:20]:
```suggestion
* Creates, applies and synchronizes local resources for your Function in the Kyma cluster

```

File: automatic/kyma-cli/README.md (hunk 3 above)
Comment (NHingerl) [inline automatic/kyma-cli/README.md:22]:
```suggestion
* [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests.

```

File: automatic/kyma-cli/README.md [hunk 8]
```diff
@@ -1,32 +1,36 @@
...
+* Runs a Function in Docker from local sources
+* [DEPRECATED] Runs and manages tests on a provisioned Kyma cluster. Using child commands, you can run tests, view test definitions, list and delete test suites, display test status, and fetch the logs of the tests
+
+![screenshot](https://cdn.jsdelivr.net/gh/dgalbraith/chocolatey-packages@ed7c4cc026f0f3a9f578dddea34adee6e38e45f0/automatic/kyma-cli/screenshot.png)
+
+## Prerequisites
+
+Kyma CLI requires the following software:
```
Comment (NHingerl) [inline automatic/kyma-cli/README.md:28]:
```suggestion
To use Kyma CLI, you must install the following software:

```

**Comment by NHingerl** [comment]
I'd suggest to crop the screenshot so it only shows the important sections, not empty screen space.

In general, I'm not sure about the purpose of the screenshot at all. 
Does it simply list the available commands? If yes, will we remember to take a new screenshot whenever the commands change?
Wouldn't it be easier to copy and paste the list as text into the document? But again, will we remember to maintain that with every change? 
So wouldn't it be easiest to link to https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma.md ?

Or is there any other reason it's here?

And is the current screenshot really up to date? It shows `completion`, `console`, `install`, `provision`, `test`, `version`, but not `deploy` and `delete`, for example. Maybe I'm misunderstanding this, but it looks outdated to me.

**Comment by shorim** [comment]
@NHingerl I think you were checking the screenshot in the master branch which is outdated and not the screenshot in my PR: https://github.com/dgalbraith/chocolatey-packages/blob/0bfb0bed7b16a3944e1f39077d7f9acc1a5acf37/automatic/kyma-cli/screenshot.png

File: automatic/kyma-cli/README.md (hunk 1 above)
Comment (shorim) [reply automatic/kyma-cli/README.md:1]: should we also update it in our CLI docs? https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma.md

File: automatic/kyma-cli/README.md (hunk 2 above)
Comment (shorim) [reply automatic/kyma-cli/README.md:9]: we are also using `which` instead of `that` in our CLI README https://github.com/kyma-project/cli/blob/main/README.md. I will update it there as well

File: automatic/kyma-cli/README.md (hunk 5 above)
Comment (shorim) [reply automatic/kyma-cli/README.md:17]: again this is how it is in our CLI docs: https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_import.md

Comment (a-thaler) [reply automatic/kyma-cli/README.md:19]: That are two separate commands, so how to do that at the same time?

**Comment by NHingerl** [comment]
> @NHingerl I think you were checking the screenshot in the master branch which is outdated and not the screenshot in my PR: https://github.com/dgalbraith/chocolatey-packages/blob/0bfb0bed7b16a3944e1f39077d7f9acc1a5acf37/automatic/kyma-cli/screenshot.png

True! My apologies, your screenshot is fine, all the right commands. Also nicely cropped :)

File: automatic/kyma-cli/README.md (hunk 5 above)
Comment (shorim) [reply automatic/kyma-cli/README.md]: We have these 2 features split into 2 different points now in the grouped features list

**Review by a-thaler — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Comment by shorim** [comment]
@dgalbraith can you take a look at this PR?

**Comment by shorim** [comment]
@dgalbraith @AnthonyMastrean @dtgm @TomOne can someone please review this PR?
We need to update the README for Kyma CLI, because the current features list is outdated.

**Comment by shorim** [comment]
@dgalbraith the conflicts are resolved now :)

**Comment by shorim** [comment]
@dgalbraith Thank you for the merge!
When should we expect the changes to be reflected in the webpage: https://community.chocolatey.org/packages/kyma-cli?

**Comment by dgalbraith** [comment]
@shorim the current version on the repository is 2.0.2.  

The version will be updated in the repository to incorporate these changes as soon as a subsequent version of Kyma CLI is released ie. Kyma CLI version >= 2.0.3.

**Comment by shorim** [comment]
@dgalbraith 
Can you please check this issue?
We even now have CLI version  2.0.4 released and still the changes in this PR are not reflected here https://community.chocolatey.org/packages/kyma-cli
